pkg/storage/stores/indexshipper/compactor/retention/retention.go

```diff
@@ -424,3 +427,50 @@ func (c *chunkRewriter) rewriteChunk(ctx context.Context, ce ChunkEntry, tableIn
 
 	return wroteChunks, linesDeleted, nil
 }
+
+// since compactor supports multiple stores, markers need to be written to store specific dir.
+// MigrateMarkers checks for markers in retention dir and migrates them.
+func MigrateMarkers(workingDir string, store string) error {
```

Comment:
Let us be more specific and call it `CopyMarkers`? Also maybe make the comment more descriptive saying we are copying markers to store specific marker directory?